vtex/affiliates-commission-service
# Feature/add variable SKU commission to order creation

#### What is the purpose of this pull request?

This PR connects our MD entities, by using the commissionBySKU service in the order creation event handler. We get the commission inside the parseData middleware.

#### How to test it?


You can use this [workspace](https://affiliates--partnerintegrationbra.myvtex.com/affiliates/lojadeteste)

- Make an order in the affiliate store
- You can see in the MD that now the SKU has a different commission than the constant 5 we used to use

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/xUPGcv4cXpq6KNcPEk/giphy.gif)

#### Types of changes
